tests/integration/templates/etcd/etcd.service

```diff
@@ -22,12 +22,12 @@
     --initial-cluster-state $CLUSTER_STATE \
     --client-cert-auth \
     --trusted-ca-file /tmp/ca-cert.pem \
-    --cert-file /tmp/etcd-cert.pem \
-    --key-file /tmp/etcd-key.pem \
+    --cert-file /tmp/client-cert.pem \
```

Comment:
Would it be possible to name these files to show they are used for the datastore? Like `datastore-client-cert.pem`?

Reply:
Hm, I mean it is the `etcd.service` file which is loaded inside the `EtcdCluster` object. So, if we want to do this it would be `etcd-client-cert.pem`, but I think that is a bit unnecessary tbh.